yaml.properties

```diff
@@ -1,15 +1,21 @@
 category=Languages
 description=Allows the analysis of YAML documents
 homepageUrl=https://github.com/sbaudoin/sonar-yaml
-archivedVersions=1.5.2,1.6.0
-publicVersions=1.7.0
+archivedVersions=1.5.2,1.6.0,1.7.0
+publicVersions=1.8.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-yaml-plugin
 
 
+1.8.0.description=Brings support for SonarQube 9.4+ and 10; adds a rule and improves some others; fixes some issues
+1.8.0.sqVersions=[8.9,LATEST]
+1.8.0.date=2023-10-23
+1.8.0.changelogUrl=https://github.com/sbaudoin/sonar-yaml/releases/tag/v1.8.0
+1.8.0.downloadUrl=https://github.com/sbaudoin/sonar-yaml/releases/download/v1.8.0/sonar-yaml-plugin-1.8.0.jar
+
 1.7.0.description=Allows rule customization from a local .yamllint file and fixes some documentation inconsistencies
-1.7.0.sqVersions=[8.9,LATEST]
+1.7.0.sqVersions=[8.9,9.3.*]
```

Comment:
Since the previous end value for this range was `LATEST`, the new value needs to be the current version. Otherwise, your users on 9.4+ won't be prompted to upgrade.

Reply:
The problem is that 1.7 does not work with SQ 9.4+